web/src/services/memoService.ts

```diff
@@ -31,7 +31,10 @@ const memoService = {
       memoFind.creatorId = userService.getUserIdFromPath();
     }
     const { data } = (await api.getMemoList(memoFind)).data;
-    const fetchedMemos = data.map((m) => convertResponseModelMemo(m));
+    let fetchedMemos: Memo[] = [];
+    if (data) {
```

Comment:
You mean the `data` could be undefined?

Reply:
Yes... I'm not sure if it's a expected behavior?

Reply:
This is certainly an unwanted value. But I don't know how it happened, it should pass error instead of returning undefined.